### PR TITLE
sglist.9: fix typo

### PR DESCRIPTION
**Pull request by daniloegea**
In the explanation section sglist_append_mbuf is duplicated. The second reference is meant to be sglist_append_single_mbuf.

### CODE REVIEW

**Review by emaste — APPROVED** [review]

**Review by mhorne — APPROVED** [review]

**Comment by mhorne** [comment]
Please, feel free to manage the commit yourself. You can add 'Approved by: mhorne (src)'.

**Comment by daniloegea** [comment]
Done, thank you, folks.

Just to be sure, if I get a PR approved here, will someone yell at me if I push to src myself (as I don't have a src commit bit)?

I didn't know we were doing that.

**Comment by mhorne** [comment]
> Done, thank you, folks.
> 
> Just to be sure, if I get a PR approved here, will someone yell at me if I push to src myself (as I don't have a src commit bit)?
> 
> I didn't know we were doing that.

No, you won't get yelled at ;)

As a freebsd project member you have sufficient privileges to push commits to the src repo, so IMO it is preferable if you handle that step.

The official policy requires "approval" for making changes to an area outside of your commit bit type:

https://docs.freebsd.org/en/articles/committers-guide/#_policy_for_committer_activity_in_other_trees

So, changes to the src tree require the review/approval of a src committer. Changes to man pages fall equally under the authority of doc committers. Basically, if someone says yes to your PR, you are fine to proceed with pushing.

**Pull request closed without merging**
